cookbooks/web/recipes/base.rb

```diff
@@ -17,13 +17,6 @@
 # limitations under the License.
 #
 
-node.set[:nfs]["/store/rails"] = {
-  :host => node[:web][:fileserver],
-  :path => "/store/rails"
-}
-
-include_recipe "nfs"
-
```

Comment:
Separating it out may well be a good idea, but it should be a separate commit really.